### ISSUE TITLE
[A11y] Desktop/Mobile Search Results Grid: Filter > Date > tick numbers

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1820]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Search

### Admin user?

No

### Issue Description

Search Results Grid: https://staging.oregondigital.org/catalog?locale=en&q%5B%5D=test&search_field=all_fields&utf8=%E2%9C%93&view=gallery

The filter section for date, the tick marks (1500-1900) under the graph needs to have more contrast for it to pass the accessibility color checker. Currently the color is color: rgb(84, 84, 84)

Instance Failed: color-contrast: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ration thresholds

Example for "1500"
Path: .flot-tick-label.tickLabel:nth-child(1)
Snippet:
`<div class="flot-tick-label tickLabel" style="position: absolute; max-width: 47px; top: 67px; left: 1px; text-align: center;">1500</div>`

### Suggested Fix

I would suggest using #575757, this value is a darker gray that passes the color contrast with a contrast ratio of 7.2:1 This is the same recommendation for the following ticket #1793 for pagination.

**Comment by CGillen** [comment]
The text color is already `rgb(84, 84, 84)`/`545454` which has better contrast than `575757`. Not sure if this was fixed somewhere and if so where.
